src/Razor/test/Microsoft.AspNetCore.Razor.Test.ComponentShim/Microsoft.AspNetCore.Components/BindMethods.cs

```diff
@@ -69,6 +69,24 @@ public static MulticastDelegate GetEventHandlerValue<T>(Func<T, Task> value)
             return value;
         }
 
+        /// <summary>
```

Comment:
The first commit is a code-dump of things that already exist in the runtime. I'm looking forward to having ref assembly generation on in the runtime repo because then we can just copy-paste those definitions here.